contrib/datacorruption/boltdb/main_test.go

```diff
@@ -76,8 +77,14 @@ func TestDropWritesDuringBench(t *testing.T) {
 		cmd := exec.Command(args[0], args[1:]...)
 		cmdCh <- cmd
 
+		var b bytes.Buffer
+		cmd.Stdout = &b
+		cmd.Stderr = &b
+
 		require.NoError(t, cmd.Start(), "start bbolt (args: %v)", args[1:])
-		waitCh <- cmd.Wait()
+		werr := cmd.Wait()
+		t.Logf("####### bbolt output: \n %s", b.String())
+		waitCh <- werr
```

Comment:
Synced @ahrtr and the case should be changed to simulate power failure instead of hardware failure.

Would you please rollback to original one and remove https://github.com/fuweid/go-dmflakey/blob/main/contrib/datacorruption/boltdb/main_test.go#L85-L94 and just sleep 3 seconds? Thanks!

Reply:
Just raised another PR https://github.com/fuweid/go-dmflakey/pull/2

Closing this PR...